.github/workflows/stable-release.yml

```
@@ -0,0 +1,35 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    branches:
+      - 'main'
+      - 'v*'
+
+    paths: 
+      - 'STABLE-RELEASE'
+
+name: KubeArmor stable release 
+
+jobs:
+  build:
+    name: Create KubeArmor Release - 18.04
```

```suggestion
    name: Create KubeArmor Release - 18.04
    if: github.repository == 'kubearmor/kubearmor'
```
Otherwise this will keep failing on individual forks.